README.md

```diff
@@ -121,12 +121,23 @@ that can be shared across all repos in an github org.
 The script assumes it will have access to the circleci variables
 
 ```sh
+# Provided by circleci. Example values shown, yours will be different.
 CIRCLE_PROJECT_USERNAME="ipfs-shipyard"
 CIRCLE_PROJECT_REPONAME="peer-pad"
 CIRCLE_SHA1="f818cb08e0e79fcc203f4d52a1a1dd7c3c832a64"
 CIRCLE_BUILD_NUMBER="1870"
 ```
 
+Optionally you can provide any of the following env vars:
+
+```sh
+# set the cluster host to pin to.
+CLUSTER_HOST="/dns/a-cluster-of-ones-own"
+
+# pass extra flags to ipfs-cluster-ctl
```

Comment:
It is only flags to the `add` subcommand.

Reply:
do you want to pass something like --debug to the top level, or is your feeling that this variable is variable should be renamed (suggestions welcome!)

Reply:
It is not big deal. I think EXTRA_IPFS_CLUSTER_ADD_ARGS might be a better name. While EXTRA_IPFS_CLUSTER_ARGS could go to top-level.